app/controllers/api/v1/rents_controller.rb

```diff
@@ -0,0 +1,25 @@
+module Api
+  module V1
+    class RentsController < ApplicationController
+      before_action :authenticate_user!
+      def index
+        user = User.find(params[:user_id])
```

Comment:
There should be a method of devise token auth that gives you the current user.

Reply:
I don't think that it be necessary for the next reason. In Apiary the path include the user id (this way user/:userid/books) so i suppose what any user authenticated can do a request for get rents info of any user.